### PR TITLE
[KHWEBA-16] About dialog

### PR DESCRIPTION
**Pull request by Konstanty-Ivashchenko**
screencapture-localhost-8080-2018-03-14-09_55_58

### CODE REVIEW

File: src/components/about-dialog/about-dialog.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  handleOk() {
+    console.log("accepted");
+  }
+
+  handleCancel() {
+    console.log("declined");
+  }
+
```
Comment (gift-a) [inline src/components/about-dialog/about-dialog.js]: You don't need this metod. You don't need button cancel too.

**Comment by gift-a** [comment]
What do you think about it?

**Comment by gift-a** [comment]
I heard smth about content.. Is it written in task?

**Comment by gift-a** [comment]
And I don't think you need ok handler too. I left it in my components for future actions.

File: src/components/about-dialog/about-dialog.js (hunk 1 above)
Comment (gift-a) [reply src/components/about-dialog/about-dialog.js]: What do you think about it?

Comment (gift-a) [reply src/components/about-dialog/about-dialog.js]: And I don't think you need ok handler too. I left it in my components for future actions.

**Comment by gift-a** [comment]
change pull request name to "about dialog"

**Review by gift-a — DISMISSED** [review]

**Comment by Konstanty-Ivashchenko** [comment]
fix

**Review by gift-a — APPROVED** [review]

**Review by OlgaOrlova — APPROVED** [review]

**Review by Vladyslav-Plakhuta — APPROVED** [review]

**Pull request merged**